resources/popup.js

```diff
@@ -1,27 +1,29 @@
+// Use the toggle button on the popup to switch the loggingEnabled option.
+function toggle() {
```

Comment:
Well.. I was changing the naming there anyway, and I thought it would be cleaner to put the function outside of the function.